templates/docker-compose.template

```diff
@@ -95,16 +90,11 @@ services:
       - miner-storage:/var/data
       - watchdog-log:/var/watchdog
       - nebra:/var/nebra
-    ports:
-      - "80:5000"
+    network_mode: host
+    extra_hosts:
+        - "helium-miner:127.0.0.1"
     cap_add:
       - SYS_RAWIO
-    devices:
```

Comment:
why are we removing these?

Reply:
Because they are unnecessary in privileged mode. They could be introduced back if that mode is left behind.